sympy/core/relational.py

```diff
@@ -1130,6 +1130,34 @@ def _eval_fuzzy_relation(cls, lhs, rhs):
     def strict(self):
         return Gt(*self.args)
 
+    def _eval_simplify(self, **kwargs):
+        # standard simplify
+        eorg = e = super()._eval_simplify(**kwargs)
```

Comment:
Is this really necessary? What is the superclass `_eval_simplify` in this case?

Reply:
I went with the similar approach as in `Equality`. I'd expect `Relational` to be the superclass.

Reply:
I didn't notice that method existed. Any reason this logic shouldn't go there?